backend/danswer/background/celery/celery_app.py

```diff
@@ -40,8 +40,6 @@
 # use this within celery tasks to get celery task specific logging
 task_logger = get_task_logger(__name__)
 
-redis_pool = RedisPool()
```

Comment:
Is there a benefit to instantiating it in the different modules? It's a singleton anyway right? Can we just do it from the one central place?

Reply:
No benefit, just how it started out. should be fine to make some sort of static or class method.